__tests__/app.test.js

```diff
@@ -347,11 +347,13 @@ describe("PATCH /api/articles/:article_id", () => {
             votes: 101,
             article_img_url:
               "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            comment_count: 11,
```

Comment:
This earlier test shouldn't be amended, and there should just be a new test to show that comment_count is now there. If the original test fails, you will need to consider using a more flexible matcher 🙂